docs/azure/integrations-overview.md

```diff
@@ -0,0 +1,360 @@
+---
+title: .NET Aspire Azure integrations overview
+description: Overview of the Azure integrations available in the .NET Aspire.
+ms.date: 12/11/2024
+uid: azure-integrations-overview
+---
+
+# .NET Aspire Azure integrations overview
+
+[Azure](/azure) is the most popular cloud platform for building and deploying [.NET applications](/dotnet/azure).  The Azure SDK for .NET allows for easy management and use of Azure services. .NET Aspire provides a set of integrations with Azure services. This article details some common aspects of all Azure integrations in .NET Aspire and aims to help you understand how to use them.
+
+## Provisioning
+
+While it's possible to provision Azure resources manually, .NET Aspire simplifies the process by providing a set of APIs to express Azure resources. These APIs are available as extension methods in .NET Aspire Azure hosting libraries, extending the <xref:Aspire.Hosting.IDistributedApplicationBuilder> interface. When you add Azure resources to your app host, they'll add the appropriate provisioning functionality implicitly. In other words, you don't need to call any provisioning APIs directly.
+
+The **Azure SDK for .NET**, provides the [📦 Azure.Provisioning](https://www.nuget.org/packages/Azure.Provisioning) NuGet package and a suite of [Azure provisioning packages](#azure-provisioning-packages). These Azure provisioning libraries make it easy to declaratively specify Azure infrastructure natively in .NET.
+
+Since .NET Aspire models Azure resources within Azure hosting integrations, the Azure SDK is used to provision these resources. The Azure SDK generates Bicep files that define the Azure resources you need. If you're unfamiliar with Bicep, it's a domain-specific language used to describe and provision Azure resources. The generated Bicep files are output alongside the manifest file when you publish your app. For more information, see [Azure provisioning Bicep](#azure-provisioning-bicep).
+
+### Azure provisioning packages
```

Comment:
I don't think we need this list so high in the overview. I think its fine to put it near the bottom with a link to the azure service as its not always obvious what it is for.